include/eve/module/bessel.hpp

```diff
@@ -20,4 +20,25 @@
 //!
 //! @}
 //==================================================================================================
-#include <eve/module/bessel/regular/bessel.hpp>
+#include <eve/module/bessel/regular/airy_ai.hpp>
+#include <eve/module/bessel/regular/airy_bi.hpp>
+#include <eve/module/bessel/regular/airy.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_i0.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_i1.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_in.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_j0.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_j1.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_jn.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_k0.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_k1.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_kn.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_y0.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_y1.hpp>
+#include <eve/module/bessel/regular/cyl_bessel_yn.hpp>
+#include <eve/module/bessel/regular/sph_bessel_j0.hpp>
+#include <eve/module/bessel/regular/sph_bessel_j1.hpp>
+#include <eve/module/bessel/regular/sph_bessel_jn.hpp>
+#include <eve/module/bessel/regular/sph_bessel_y0.hpp>
+#include <eve/module/bessel/regular/sph_bessel_y1.hpp>
+#include <eve/module/bessel/regular/sph_bessel_yn.hpp>
+//#include <eve/module/bessel/regular/bessel.hpp>
```

Comment:
remove don't comment

Reply:
it seems to me that this file has already been updated